seed/views/v3/properties.py

```diff
@@ -45,3 +208,767 @@ def labels(self, request):
+
+        return JsonResponse(response)
+
+    @swagger_auto_schema(
+        manual_parameters=[
+            AutoSchemaHelper.query_org_id_field(),
+            AutoSchemaHelper.query_integer_field(
+                'cycle',
+                required=False,
+                description='The ID of the cycle to get properties'),
+        ],
+        request_body=AutoSchemaHelper.schema_factory(
+            {
+                'profile_id': 'integer',
+                'inventory_ids': ['integer'],
+            },
+            required=['profile_id'],
+            description='Properties:\n'
+                        '- profile_id: Either an id of a list settings profile, or undefined\n'
+                        '- inventory_ids: List of inventory ids'
+        )
+    )
+    @api_endpoint_class
+    @ajax_request_class
+    @has_perm_class('requires_viewer')
+    @action(detail=False, methods=['POST'])
+    def filter(self, request):
+        """
+        List all the properties
+        """
+        if 'profile_id' not in request.data:
+            profile_id = None
+        else:
+            if request.data['profile_id'] == 'None':
+                profile_id = None
+            else:
+                profile_id = request.data['profile_id']
+
+                # ensure that profile_id is an int
+                try:
+                    profile_id = int(profile_id)
+                except TypeError:
+                    pass
+
+        return self._get_filtered_results(request, profile_id=profile_id)
+
+    @swagger_auto_schema(
+        manual_parameters=[
+            AutoSchemaHelper.query_array(
+                'inventory_ids',
+                item_type='integer',
+                required=True,
+                description='Array containing Property IDs.'
+            )
+        ]
+    )
+    @api_endpoint_class
+    @ajax_request_class
+    @action(detail=False, methods=['GET'])
+    def meters_exist(self, request):
+        """
+        Check to see if the given Properties (given by ID) have Meters.
+        """
+        property_ids = request.query_params.get('inventory_ids', [])
+
+        return Meter.objects.filter(property_id__in=property_ids).exists()
+
+    @swagger_auto_schema(
+        manual_parameters=[AutoSchemaHelper.query_org_id_field()],
+        request_body=AutoSchemaHelper.schema_factory(
+            {
+                'state_ids': ['integer']
+            },
+            required=['state_ids'],
+            description='Array containing property state ids to merge'),
+    )
+    @api_endpoint_class
+    @ajax_request_class
+    @has_perm_class('can_modify_data')
+    @action(detail=False, methods=['POST'])
+    def merge(self, request):
+        """
+        Merge multiple property records into a single new record, and run this
+        new record through a match and merge round within it's current Cycle.
+        """
+        body = request.data
+
+        state_ids = body.get('state_ids', [])
```

Comment:
@adrian-lara @aviveiros11 I missed this, it looks like we _do_ use property state ids in a body for this request. Maybe we should clarify that path ids are property view ids. Thoughts?

Reply:
I think we should convert these to view_ids. Check out https://github.com/SEED-platform/seed/pull/2257/commits/cb17d404b62964667702712699d70e02d4165f85